### PR TITLE
[FSDE-13216] Added support of one-color Marketing Icon

### PR DESCRIPTION
**Pull request by janibolkvadze**

### CODE REVIEW

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-305/1/. :x:

File: src/custom-elements/docs/docs.json [hunk 1]
```diff
@@ -2088,6 +2088,28 @@
           "optional": false,
           "required": false
         },
+        {
+          "name": "mode",
```
Comment (mattnickles) [inline src/custom-elements/docs/docs.json:2092]: @janibolkvadze @jllr Thought on using `variant` instead of mode? We use variant on the button component.

**Review by mattnickles — APPROVED** [review]
Default color of the one-color icons should be defined in the CSS and it needs to be themeable. I will send a PR

**Comment by mattnickles** [comment]
> Default color of the one-color icons should be defined in the CSS and it needs to be themeable. I will send a PR

Will also include theme support full-color. They appear to no longer be rendering correctly when changing themes.

File: src/website/views/components/marketing-icon/_examples.pug [hunk 2]
```diff
@@ -1,3 +1,10 @@
...
...'products-manufacturing','products-media-and-entertainment','products-migration','products-mpls-ip-vpn-service','products-news','products-online-shopping','products-pharma','products-plane-satellite-internet','products-portal','products-private-branch-exchange','products-private-fiber-network','products-retail','products-route-miles','products-router','products-sap-applications','products-satellite','products-screen-sharing','products-sd-wan','products-sd-wan-router-intelligent-routing','products-security','products-security-endpoint-application','products-services','products-session-border-controllers','products-ship-satellite-internet','products-sip-voice-calling','products-speed','products-system-integrators','products-tech-support','products-teleport','products-threat-detect','products-threat-intelligence','products-total-solution','products-train-satellite-internet','products-uc-and-c','products-ui-ux-modernization','products-upload-download','products-video-chat','products-virtual-desktop','products-voice','products-vpn-gateway','products-vulnerability','products-warning','products-wavelength','products-web-application-firewall','products-wifi', 'products-network-solutions'];
```
Comment (mattnickles) [inline src/website/views/components/marketing-icon/_examples.pug:6]: I will also include a fix for this - `products-network-solutions` isn't ordered alphabetically

**Review by jllr — APPROVED** [review]

**Review by jllr — APPROVED** [review]

**Pull request merged**